CFC_DataCollector/recommender/augment_trips_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import get_trips
+#import Profiles
+
+def augment_trips(trip_list):
+    '''
+    store trips to database
+    '''
+    return    
+
+def get_bare_trips(user_ID, filter_queries):
+    '''
+    get a list of user trips, using get_trips logic
+    
+    '''
+    return
+
+def get_user_ids(filter_queries):
```

Comment:
Again, who is expected to pass in the filter_queries to the pipeline? Is this the same filter_queries as the previous one?